src/lib.rs

```diff
@@ -420,7 +419,7 @@ impl<E: Display> Display for AllocOrInitError<E> {
 ///
 /// Even when optimizations are on, these functions do not **guarantee** that
 /// the value is constructed on the heap. To the best of our knowledge no such
-/// guarantee can be made in stable Rust as of 1.44.
+/// guarantee can be made in stable Rust as of 1.51.
```

Comment:
I changed this to 1.51, since that is the new MSRV, I assume this is still true?